### DISCUSSION TITLE
Addons / Plugins in a self hostet Version?

### DISCUSSION THREAD

**Discussion by OscarWilde** [codex-team/editor.js#2652] [Q&A] [answered]
Hi all,

now I know that I can include editorjs.umd.js in a normal Script-Tag like `<script src ="[js/editorjs/editorjs.umd.js" type="text/javascript"></script>`

But how do I get the additional features like 'Header' or 'Image'?

Thanks in advance! Frank

**Comment by neSpecc** [comment, marked as answer]
We don't have Releases in tools at the moment. You can get bundles from npm or JSDelivr.

Example for Warning Tool:
NPM: https://www.npmjs.com/package/@editorjs/warning?activeTab=code
JsDelivr https://www.jsdelivr.com/package/npm/@editorjs/warning?tab=files
